packages/trader/src/index.tsx

```diff
@@ -1,8 +1,3 @@
-// import 'babel-polyfill';
-import 'promise-polyfill';
-
-import 'event-source-polyfill';
```

Comment:
Please remove `event-source-polyfill` from the core package too. We don't use it anywhere in our app.

Reply:
Done.